src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -12054,11 +12055,13 @@ void I1.M1()
             {
                 var compilation3 = CreateCompilation(source2, new[] { reference }, options: TestOptions.DebugExe,
                                                      parseOptions: TestOptions.Regular,
-                                                     targetFramework: TargetFramework.StandardLatest);
+                                                     targetFramework: TargetFramework.Net50);
 
                 compilation3.VerifyDiagnostics();
 
+#if NETCOREAPP
```

Comment:
@AlekseyTs wanted to make sure you reviewed both uses of `#if NETCOREAPP` in this change. These `CompileAndVerify` calls are executing DIM methods and that should only be legal on .NET Core. Yet this was running on .NET Framework as well. I'm not quite sure how it was running before my change as that shouldn't work.

This change _feels_ correct but I'm also hesitant as it was running before so I wanted to make sure you looked this part over.